packages/repository/test/unit/decorator/model-and-relation.decorator.unit.ts

```diff
@@ -7,7 +7,7 @@ import {MetadataInspector} from '@loopback/context';
 import {expect} from '@loopback/testlab';
 import {RelationMetadata} from '../../..';
 import {
-  belongsTo,
+  belongsToUniquely,
```

Comment:
I think we should preserve existing tests for `belongsTo` and create new tests verifying specific differences of `belongsToUniquely`. Thoughts?